.github/workflows/conventional-commits.yaml

```
@@ -0,0 +1,16 @@
+name: Conventional Commits
+
+on:
+  pull_request:
+    branches: [ 'main' ]
+
+jobs:
+  build:
+    name: Conventional Commits
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)